platform/viewer/cypress/integration/common/OHIFRoutes.spec.js

```diff
@@ -0,0 +1,42 @@
+// type definitions for Cypress object "cy"
+/// <reference types="cypress" />
```

Comment:
Why typedef comments?

Reply:
This typedef automatically activates an Intelligent Code Completion [(check it here)](https://docs.cypress.io/guides/tooling/intelligent-code-completion.html#Writing-Tests). We can remove it if it doesn't make sense to keep it. (but is pretty cool :) )